### PR TITLE
fix payload not logged by some services when using log middleware

### PR DESCRIPTION
**Pull request by cikupin**
## What does this PR do?
fix payload not logged by some services when using log middleware

## Why are we doing this? Any context or related work?
https://kitabisa.atlassian.net/browse/CIA-663

## Where should a reviewer start?
_optional -- if your changes affected so much files, it is encouraged to give helper for reviewer_

## Screenshots
_optional -- You may put the database, sequence or any diagram needed_

## Manual testing steps?
1. use `RequestLogger` middleware
2. if any error happened, payload body should be logged along with other request informations

### CODE REVIEW

File: middleware/log.go [hunk 1]
```diff
@@ -33,6 +33,21 @@ func RequestLogger(next http.Handler) http.Handler {
 	fn := func(w http.ResponseWriter, r *http.Request) {
 		ww := cmiddleware.NewWrapResponseWriter(w, r.ProtoMajor)
 
+		var body string
+		if !strings.Contains(r.Header.Get("Content-type"), "multipart/form-data") {
```
Comment (pushm0v) [inline middleware/log.go:37]: Why this validation exclude "multipart/form-data" ? What happen if content type is binary ?

**Review by ardityawahyu — APPROVED** [review]

File: middleware/log.go (hunk 1 above)
Comment (ardityawahyu) [reply middleware/log.go:37]: @pushm0v because it can be too big to log, especially when there's file uploaded

Comment (pushm0v) [reply middleware/log.go:37]: agreed, but the validation will pass for `application/octet-stream` or `application/x-binary`

Comment (ardityawahyu) [reply middleware/log.go:37]: @pushm0v ah yeah, didn't thing that we will use those content-type. @cikupin maybe we can include those two too.

Comment (pushm0v) [reply middleware/log.go:37]: or the other around is to check the body length, if bigger than X bytes, drop it off

Comment (cikupin) [reply middleware/log.go]: updated. excluding application/octet-stream and application/x-binary

Comment (ardityawahyu) [reply middleware/log.go]: @cikupin may consider about the body length like mas @pushm0v mentioned above?

Comment (cikupin) [reply middleware/log.go]: kalo untuk upload file, gw prefer gk ngelog sih. karna kalo di print nanti bakal jadi kayak semacam string hash yang panjang gitu. jadinya di log bakal panjang banget

Comment (ardityawahyu) [reply middleware/log.go]: setuju soal itu. Cuma handling nya instead dilimit pake content-type, gimana klo filtering nya pake body length nya? jadi klo misal exceed 1MB, ngga perlu dilog. @cikupin

Comment (cikupin) [reply middleware/log.go]: @ardityawahyu updated mas

**Review by ardityawahyu — APPROVED** [review]

**Review by budiariyanto — APPROVED** [review]

**Pull request merged**